common/styles/scss/app.scss

```diff
@@ -723,3 +723,20 @@ html {
         line-height: 1.2;
     }
 }
+
+.top-scroll-wrapper {
```

Comment:
Given that app.css is included in the static pages that have navbar, we have to make sure our rules are properly scoped and also use specific names if we can.

In this case, let's move these two new rules that you added to be in inside the `.chaise-body` wrapper above. Also let's just change the class names that you added to have `chaise-` and also more specific to table, so

- `top-scroll-wrapper` -> `chaise-table-top-scroll-wrapper`
- `top-scroll` -> `chaise-table-top-scroll`

Reply:
Makes sense, made the necessary changes